The Data Carrier (New) - The Grand Arcadia

AjexRose opened the pull request:
<!-- Please read these guidelines before opening your PR: https://docs.spacestation14.io/en/getting-started/pr-guideline -->
<!-- The text between the arrows are comments - they will not be visible on your PR. -->

## About the PR
<!-- What did you change in this PR? -->
after finally getting off my ass, I decided to finally finish this, this PR adds a new bluespace event known as the "Arcadia Industries Data Carrier" A bluespace event within a fully operational shuttle, filled with lots of loot and entities to fight though. Also includes changes to the Arcadia Industries that were added awhile ago, mainly replacing their blood with oil, and making the Mobile Blaster Units space proof. 

## Why / Balance
<!-- Why was it changed? Link any discussions or issues here. Please discuss how this would affect game balance. -->
Gives Arcadia Industries more visibility then just a vending machine. 
Edit: Also forgot to mention most of the crates are simple random low value spawners.
The high amount of loot within is countered by the fact that the shuttle is simply bursting from the seems with enemies, all ready to pounce at intruders.
The main focus is the R&D loot, which there isnt much attention for as of now in the Frontier. 
## Technical details
<!-- If this is a code change, summarize at high level how your new code works. This makes it easier to review. -->
N/A
## Media
<!-- 
PRs which make ingame changes (adding clothing, items, new features, etc) are required to have media attached that showcase the changes.
Small fixes/refactors are exempt.
Any media may be used in SS14 progress reports, with clear credit given.

If you're unsure whether your PR will require media, ask a maintainer.

Check the box below to confirm that you have in fact seen this (put an X in the brackets, like [X]):
-->

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase
![Frontier Station 1_23_2024 2_34_38 PM](https://github.com/new-frontiers-14/frontier-station-14/assets/112997230/49560bf6-a3ee-455d-8fd0-f42f5a6e7d72)
![Frontier Station 1_23_2024 2_31_25 PM](https://github.com/new-frontiers-14/frontier-station-14/assets/112997230/843c07ec-fc70-4e60-bc8a-52e32f02fb5c)

## Breaking changes
<!--
List any breaking changes, including namespace, public class/method/field changes, prototype renames; and provide instructions for fixing them. This will be pasted in #codebase-changes.
-->

**Changelog**
<!--
Make players aware of new features and changes that could affect how they play the game by adding a Changelog entry. Please read the Changelog guidelines located at: https://docs.spacestation14.io/en/getting-started/pr-guideline#changelog
-->

:cl: AjexRose
- add: Added a new bluespace event: The Data Carrier, with lots of loot, enemies, and a small focus on R&D
- tweak: Arcadia Industries mobs have been slightly reworked to no longer bleed blood.
- tweak: Fixed the ArcadiaDrobe's vendor sprite

FoxxoTrystan commented on Resources/Prototypes/_NF/Events/events.yml:
```diff
@@ -140,3 +140,22 @@
+    startAnnouncement: station-event-bluespace-ArcIndDataCarrier-start-announcement
+    startAudio:
+      path: /Audio/Announcements/attention.ogg
+    endAnnouncement: station-event-bluespace-ArcIndDataCarrier-end-announcement
```
Same here, prefer to use "station-event-bluespace-ship-end-announcement"

FoxxoTrystan reviewed (changes requested):
Remove Random Magic Book Spawner (Or at least use the safe one)

Recomendation/Notes:
* Wizard hardsuit seem compleatly out of place (but also because there was plan to put those in more wizard/alien looking asteroids)
* Atmos is ehh, started then gone?
* Floor need decals in my opinion a bit.
* Does your NPCs has the salvage component? if no they need it so they despawn when the ship despawn.
* Again make the ship name/annoncement to the default/random.

AjexRose commented on Resources/Maps/_NF/Bluespace/DataCarrier.yml:
```diff
@@ -0,0 +1,9938 @@
+  entities:
+  - uid: 1
+    components:
+    - name: AI-DC01-003
```
Done

AjexRose commented:
After testing it via addgamerule and endgamerule, the mobs despawn without the salvage component when the event ends. Either that or they already have it somewhere.

AjexRose commented:
also the spell book was set to safe, and the wizard hardsuit was replaced by syndicate hardsuit

FoxxoTrystan commented on Resources/Locale/en-US/_NF/bluespace-events/events.ftl:
```diff
@@ -9,3 +9,4 @@ station-event-bluespace-asteroid-end-announcement = In compliance with NanoTrase
 
 station-event-bluespace-ship-start-announcement = We have detected an unusual FTL signature - long range scans indicate an unknown ship. NanoTrasen cannot confirm safety for prospectors within its vicinity, be advised.
 station-event-bluespace-ship-end-announcement = In compliance with NanoTrasen FTL traffic patterns, the unknown ship has been dissipated to ensure non-collision.
+
```
Hmmm Space!

FoxxoTrystan reviewed (commented):
All good, tho the Syndicate Hardsuit is an Elite one and seem a bit too easy to get in my opinion but overall il say generally its fine.

AjexRose commented:
Updated to have more decals, removed the omega soap, replaced the syndicate hardsuit with AME fuel, switched power from damaged RTGs to regular RTGs, replaced the unused atmos space with a space for portable scrubbers.

Cheackraze commented:
> Updated to have more decals, removed the omega soap, replaced the syndicate hardsuit with AME fuel, switched power from damaged RTGs to regular RTGs, replaced the unused atmos space with a space for portable scrubbers. Also changed the file name to match current naming standards

was just looking to do some map reviews and such. Do you have any pictures of the new decaling? it was going to be my only critique of the map itself. Otherwise, I can help update this to the latest build and get this in game, however the vending machine sprite needs fixing desperately as well, if you wouldnt mind taking a look at cleaning that one up.

Cheackraze commented:
just tested it out a few times, the map itself looks nice, especially with the added decals. I'm only a little concerned that it will get powergamed everytime and might in the future re-arrange what items are stored in each of your vault areas. If we could just get that vending machine sprite cleaned up i'd say its good to merge

AjexRose commented:
Just got home from uni, so apologies for not getting back to you sooner
> just tested it out a few times, the map itself looks nice, especially with the added decals. I'm only a little concerned that it will get powergamed everytime
Most items are simple fill-in substitutes until I get more faction unique items made.
> If we could just get that vending machine sprite cleaned up i'd say its good to merge
I'll start to work on that ASAP

AjexRose commented:
vending machine's display sprite has been fixed at last!
